src/Controller/UsersController.ts

```diff
@@ -11,19 +11,21 @@ import {
 import TYPES from '../Bootstrap/Types'
 import { GetSetting } from '../Domain/UseCase/GetSetting/GetSetting'
 import { GetSettings } from '../Domain/UseCase/GetSettings/GetSettings'
+import { GetUserKeyParams } from '../Domain/UseCase/GetUserKeyParams/GetUserKeyParams'
 import { UpdateUser } from '../Domain/UseCase/UpdateUser'
 
-@controller('/users', TYPES.AuthMiddleware)
+@controller('/users')
 export class UsersController extends BaseHttpController {
   constructor(
     @inject(TYPES.UpdateUser) private updateUser: UpdateUser,
     @inject(TYPES.GetSettings) private doGetSettings: GetSettings,
     @inject(TYPES.GetSetting) private doGetSetting: GetSetting,
+    @inject(TYPES.GetUserKeyParams) private getUserKeyParams: GetUserKeyParams,
   ) {
     super()
   }
 
-  @httpPatch('/:userId')
+  @httpPatch('/:userId', TYPES.AuthMiddleware)
```

Comment:
now it makes sense! ;)

Reply:
:)

Reply:
some context: https://github.com/inversify/inversify-express-utils#basemiddleware